.circleci/config.yml

```diff
@@ -37,6 +37,20 @@ jobs:
             cd ../k8s-rhel7
             make build push
 
+  build_k8s_bundles:
+    machine: true
+      docker_layer_caching: true
```

Comment:
should we enabled this for the other docker builds as well?

Reply:
yes, makes sense